### ISSUE TITLE
Variables from useDefaultOptions are not passed to queryFn

### ISSUE THREAD

**Issue by SpadarShut** [liaoliao666/react-query-kit#18]
Variables from useDefaultOptions are actually never passed to queryFn. 

```js
const usePost = createQuery<Response, Variables, Error>({
  primaryKey: '/posts',
  queryFn: ({ queryKey: [primaryKey, variables] }) => {
    // variables is actually undefined here
    // ... 
  },
  useDefaultOptions: () => ({ variables: { id: 42 }})
})
```

Passing variables to hook instance works however.

**Comment by liaoliao666** [comment]
Already fixed in v1.4.3

**Comment by SpadarShut** [comment]
Thanks!
